index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module 'queue-fifo' {
+    class Queue<T> {
```

Comment:
You might be able to get away with:
```typescript
export declare class Queue<T> {
...
}
```

It may save you the declared module name, but this will work, too. :smile:

Reply:
Thanks for the help and review, @CommanderQ!

Reply:
Thanks again, @thieman for taking the time and interest to submit the PR.  Much appreciated and well done. 👍 

Fix #1